vmware/vmware-cloud-director-ui-components
# Ng11 upgrade

## PR Checklist

Please check if your PR fulfills the following requirements:


## PR Type
Upgrade

## What does this change do?
Upgrades to Angular 11 and Clarity 5.

## What manual testing did you do?
Running the examples app. 

* Check each example in the examples application.
* Verify API tabs and execute each example for each component observing its described behavior.
* Stackblitz examples are broken. Created internal issue https://jira.eng.vmware.com/browse/VDUCC-739

## Does this PR introduce a breaking change?
